manifests/worker.pp

```diff
@@ -123,14 +140,12 @@
         'encrypt'          => $consul_encrypt,
         'ui'               => $consul_ui,
       },
-      services    => {
-        'mesos-slave' => {
-          port => 5051
-        },
-      },
       require     => Package['unzip'],
     }
   }
+  consul::service { 'mesos-slave':
+    port => 5051
+  }
```

Comment:
You're right... Consul doesn't start in client mode with `bootstrap_expect` set. Kind of a separate issue but since it's a small change I may as well fix here...